.gitmodules

```diff
@@ -193,3 +190,6 @@
 [submodule "community-build/community-projects/cask"]
 	path = community-build/community-projects/cask
 	url = https://github.com/dotty-staging/cask.git
+[submodule "community-build/community-projects/psbp"]
+	path = community-build/community-projects/psbp
+	url = https://github.com/PSBP-Library/psbp-library.github.io.git
```

Comment:
I think we should keep pointing to our fork, but I didn't follow the latest community build changes...
wdyt @anatoliykmetyuk?

Reply:
Yes, that’s right. I created a fork: https://github.com/dotty-staging/psbp-library.github.io
And invited @PSBP-Library there. Please point to this repo.